bakery/service/parser.go

```diff
@@ -57,21 +57,26 @@ func (dip *dockerImageParser) ParseDockerfile(dockerfilePath string) (*DockerIma
 
 		dependsFromLong := strings.TrimPrefix(line, dependencyPrefix)
 		dependsFromShort := dependsFromLong
+		dependsFromVersion := ""
```

Comment:
What would you say about renaming those variables to start with `dependsOn...` - this sounds more natural to me.
Also, https://en.oxforddictionaries.com/definition/depend  says the current version is archaic...

Reply:
No problem, will do that.